main.go

```diff
@@ -996,9 +997,44 @@ func adminOnly(h http.HandlerFunc) http.HandlerFunc {
 }
 
 // Handle individual topic operations
+// handleUserExerciseStats handles requests for user's exercise statistics.
+func handleUserExerciseStats(w http.ResponseWriter, r *http.Request) {
+	w.Header().Set("Content-Type", "application/json")
+	w.Header().Set("Access-Control-Allow-Origin", "*")
+
+	if r.Method == http.MethodOptions {
+		w.WriteHeader(http.StatusOK)
+		return
+	}
+
+	cookie, err := r.Cookie("user_id")
```

Comment:
do we just trust id in cookie? what if a molestor will set someone's id into their cookie?

I think this problem might be in other auth endpoints